CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * Added the `--use-pack-metadata` flag for the **find-dependencies** command to update the calculated dependencies using the the packs metadata files.
 * Fixed an issue where **validate** failed on scripts in case the `outputs` field was set to `None`.
 * Fixed an issue where **validate** was failing on editing existing release notes.
+* Added a validation for README files verifying that the file doesn't contain template text copied from HelloWorld or HelloWorldPremium README.
```

Comment:
you've added a check for empty pack readmes here, please add a note for that as well.

Reply:
Done